README.md

```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
 
 You start by working through our self-paced, browser-based full stack JavaScript curriculum.
 
-### By working through our curriculum, you can earn four certifications:
-##### 1. Front End Certification
-The first section will teach you the basics of how webpages work and also introduce you to JavaScript programming.
+### By working through our new and updated curriculum and completing the required projects, you can earn six certifications. Each of these certificates will have 5 required projects, and all other challenges will be optional.      We estimate that each certificate will take around 300 hours to complete.
 
-Skills you'll practice include `HTML`, `CSS`, `JavaScript`, `jQuery` and `Bootstrap`.
 
-To earn this certification, you'll build **10 front-end projects** and implement many **JavaScript algorithms**.
+##### 1. Responsive Web Design
 
-##### 2. Data Visualization Certification
-The second section builds upon the first and introduces you to more advanced topics such as `Sass`, `React` and `D3`.
+**Add links to the live demos of your projects as you finish them. Then, once you have added all 5 projects required for a certification, you can claim it.**
+a. Build A Tribute Page
```

Comment:
No need to label the challenges, we can just bullet-point them
```md
- item one
- item two
- item three
```
Becomes 
- item one
- item two
- item three

That is to say, they are all of equal importance